eZ/Publish/API/Repository/Values/User/Role.php

```diff
@@ -38,6 +55,41 @@
     protected $identifier;
 
     /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT|Role::STATUS_MODIFIED
+     */
+    protected $status;
```

Comment:
@andrerom Yes, but RoleDraft (as based on ContentTypeDraft) is just a wrapper around a Role object, and doesn't have any other properties. Isn't it good to be able to as a Role object that status it has? Particularly later if we are going to use STATUS_MODIFIED for e.g. deferred cache clearing?